core/arch/x86/decode_private.h

```diff
@@ -345,6 +345,7 @@ struct _decode_info_t {
     /* for instr_t* target encoding */
     ptr_int_t cur_note;
     bool has_instr_opnds;
+    byte tuple_type;
```

Comment:
Shouldn't this be the enum type?

Reply:
I wanted to limit its size, but yes. I've changed it to dr_tuple_type_t.

Reply:
For limiting size, have a comment explaining what the type is, like we do elsewhere, so it's clear:
```C
            byte /*dr_extend_type_t*/ extend_type : 3;
```